cogs/qrchecker/qrchecker.py

```diff
@@ -28,14 +31,31 @@ class QRChecker(commands.Cog):
     def __init__(self, bot: Red):
         self.bot: Red = bot
         self.logger: Logger = getLogger("red.luicogs.QRChecker")
+        self.lock: Lock = Lock()
         self.config: Config = Config.get_conf(self, identifier=5842647, force_registration=True)
         self.config.register_guild(**BASE_GUILD)
+        self.config.register_global(**BASE_GLOBAL)
+        self.initialized: bool = False
+        self.bgTask = self.bot.loop.create_task(self.setMaxImagePixels())
+
+    async def setMaxImagePixels(self):
+        pixels = await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS)()
+        self.logger.debug("Setting max pixels to %s", pixels)
+        Image.MAX_IMAGE_PIXELS = pixels
+        self.initialized = True
```

Comment:
I know the context here is that we set max image pixels in cog init stage but this is quite misleading if we look at the code at some point in the future.

How about:
- Having a method for init steps and `setMaxImagePixels()` is included in it?
- Refactoring `setMaxImagePixels` so that it can be used by command handling logic as well?

```suggestion
    def __init__(self, bot: Red):
        self.bot: Red = bot
        self.logger: Logger = getLogger("red.luicogs.QRChecker")
        self.lock: Lock = Lock()
        self.config: Config = Config.get_conf(self, identifier=5842647, force_registration=True)
        self.config.register_guild(**BASE_GUILD)
        self.config.register_global(**BASE_GLOBAL)
        self.initialized: bool = False
        self.bgTask = self.bot.loop.create_task(self.init())

    async def init(self):
        maxImagePixels = await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS)()
        await self.setMaxImagePixels(value=maxImagePixels)
        self.initialized = True

    async def setMaxImagePixels(self, value: Optional[int] = None):
        """Set PIL/Pillow's max image pixels to prevent `DecompressionBombWarning`.
        If `value` is `None`, the max image pixels value from config will be used."""

        if value is None:
            value = await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS)()
        else:
            await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS).set(value)
        Image.MAX_IMAGE_PIXELS = value
        self.logger.debug("Set max pixels to %s.", value)
```